packages/solana/src/client.ts

```diff
@@ -387,8 +389,10 @@ export class Web3Modal extends Web3ModalScaffold {
 
     const uniqueIds = standardAdapters ? new Set(standardAdapters.map(s => s.name)) : new Set([])
     const filteredAdapters = this.walletAdapters.filter(
-      adapter => !uniqueIds.has(adapter.name) && uniqueIds.add(adapter.name)
+      adapter =>
+        (!uniqueIds.has(adapter.name) || adapter.name === 'Trust') && uniqueIds.add(adapter.name)
```

Comment:
Why `Trust` in this list? Could you create a enum for this with a clear name? Like let's say it should be filtered out, then like;

```js
const FILTER_OUT_ADAPTERS = ["Trust"]
```

and use this;

```js
... FILTER_OUT_ADAPTERS.includes(adapter.name) ...
```

This will keep our code more maintainable.